src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    private static final Random RANDOM = new Random();
+
+    public Colors getRandomColor() {
+        int index = new Random().nextInt(Colors.values().length);
```

Comment:
You're creating a new instance of Random every time the method is called, which is unnecessary since you already have a Random instance defined as a constant on line 6. Use the existing RANDOM constant to call nextInt.